client/src/debug/FabricDebugConfigurationProvider.ts

```diff
@@ -50,26 +51,26 @@ export abstract class FabricDebugConfigurationProvider implements vscode.DebugCo
                 return;
             }
 
-            if (!this.runtime.isDevelopmentMode()) {
+            const developmentMode: boolean = await this.runtime.isDevelopmentMode();
```

Comment:
i think we can take this out. We need to update the generator version anyway so if we update the version here then they will have to tear down anyway and then always be in dev mode

Reply:
Removed all signs of development mode!